internal/shim-sev/src/syscall.rs

```diff
@@ -196,7 +199,7 @@ pub fn syscall_rust(
         // Enarx syscalls
 
         // Fixme
-        let ret: sallyport::Result = Err(libc::ENOSYS);
+        let ret: sallyport::Result = Ok([0.into(), SEV_TECH.into()]);
```

Comment:
```suggestion
        let ret: sallyport::Result = match nr {
            0xEA01 => Ok([0.into(), SEV_TECH.into()]),
            _ => Err(libc::ENOSYS),
        };
```

SGX has a `SYS_GETATT` but I don't see one defined for SEV.